CITATION.cff

```diff
@@ -0,0 +1,248 @@
+cff-version: 1.2.0
+message: If you use this standard or software, please cite it using the metadata from this file.
+title: Machine Learning Model Extension Specification for SpatioTemporal Asset Catalog
+type: software
+keywords:
+  - mlm
+  - Machine Learning
+  - Model
+  - STAC
+repository: "https://github.com/crim-ca/mlm-extension"
```

Comment:
I see this and one other link to the old repository that isn't getting updates anymore. I'm not sure how this CITATION.cff gets rendered in github, do you know @fmigneault ? I think it'd be preferable to have the citation that a user copies point to this current repository since this is the standard they are using and it is better for visibility of the current standard.

Reply:
I'll move this to a separate reference as the "old version".